ubuntu-apache-test-page/azuredeploy.json

```diff
@@ -0,0 +1,286 @@
+﻿{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "adminUsername": {
+      "type": "string",
+      "minLength": 1,
+      "metadata": {
+        "description": "User name for the Web Server VM."
+      }
+    },
+    "adminPassword": {
+      "type": "securestring",
+      "metadata": {
+        "description": "Password for the Web Server VM."
+      }
+    },
+    "dnsNameForPublicIP": {
+      "type": "string",
+      "minLength": 1,
+      "metadata": {
+        "description": "Globally unique DNS Name for the Public IP used to access the Web Server VM."
+      }
+    },
+    "ubuntuOSVersion": {
+      "type": "string",
+      "defaultValue": "14.04.5-LTS",
+      "allowedValues": [
+        "14.04.5-LTS",
+        "16.04.0-LTS"
+      ],
+      "metadata": {
+        "description": "The Ubuntu version for the Web Server VM. Allowed values: 12.04.5-LTS, 14.04.5-LTS, 16.04.0-LTS"
+      }
+    },
+    "testPage": {
+      "type": "string",
+      "defaultValue": "index.html",
+      "metadata": {
+        "description": "Test page you want to create on the Web Server."
+      }
+    },
+    "testPageTitle": {
+      "type": "string",
+      "defaultValue": "Test Page",
+      "metadata": {
+        "description": "Test page title."
+      }
+    },
+    "testPageBody": {
+      "type": "string",
+      "defaultValue": "<p>This is a test page.</p>",
+      "metadata": {
+        "description": "Test page content body markup."
+      }
+    },
+    "installPHP": {
```

Comment:
Is this needed for the sample?  If I change it, what are all the other changes I would need to be aware of?